src/App.js

```diff
@@ -25,9 +24,10 @@ function App() {
       <section>
       <Routes>
         <Route path="/" element={<Articles />} />
-        <Route path="/:topic" element={<Articles />} />
+        <Route path="/articles/topics/:topic" element={<Articles />} />
         <Route path="/articles/:article_id" element={<SingleArticle />} />
         <Route path="/users" element={<ChangeUsers />} />
+        <Route path="*" element={<h3>No path found 🐬 🐬 🐬</h3>} />
```

Comment:
Any emojis we use should be wrapped in a <span> tag and given an attribute of role="img" for accessibility reasons - I'm getting a console warning about this so you might want to fix it before deploying your app